include/openssl/comp.h.in

```diff
@@ -1,12 +1,20 @@
 /*
- * Copyright 2015-2018 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2024 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Copyright 4 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright (c) 2002, Oracle and/or its affiliates. All rights reserved
+ * Copyright 2005 Nokia. All rights reserved.
```

Comment:
I wouldn't add this. None of the things moved were IMO added by these copyright holders.